src/core_ocean/shared/mpas_ocn_tracer_advection_mono.F

```diff
@@ -701,9 +701,9 @@ subroutine ocn_tracer_advection_mono_tend(tend, tracers, layerThickness,    &
           do iCell = 1, nCells
             kmax = maxLevelCell(iCell)
             do k=3,kmax-1
-              verticalWeightK   = hProv(k-1,iCell) / &
+              verticalWeightK   = hProv(k  ,iCell) / &
```

Comment:
@vanroekel the original version was correct on all these. It is not a weighted average, it is an interpolation.
- weighted average: ( h_k T_k + h_{k-1} T_{k-1} ) / ( h_k + h_{k-1})
- interpolation: ( h_{k-1} T_k + h_k T_{k-1} ) / ( h_k + h_{k-1})

![image](https://user-images.githubusercontent.com/4036816/97655337-76434200-1a2a-11eb-8d0f-22d1a4c0fa1b.png)
It seems opposite. I saw this in 2017 and thought it was an error too. But for interpolation, as a layer thickens (h_k) the value at the interface gets closer and closer to the value in the _opposite_ layer (h_{k-1}). For the equation of a line, you would actually have factors of 1/2 on all the thicknesses, but those all divide out.

Reply:
ahh, thanks @mark-petersen!  That makes sense now.  I'll take those out.